app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` có tác dụng gì em

Reply:
`user&.authenticate` tức là `user && user.authenticate`
`&&` là toán tử logic `and `
Ở đây, điều kiện này đang kiểm tra user tồn tại và đúng mật khẩu hay không ạ. Nếu user `nil` thì điều kiện trả về là `nil`